src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  border: 1px solid $color-gray;
+  padding: 32px 16px 16px;
+
+  &__description {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $main-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+    margin-top: 16px;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    color: $secondary-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__total {
+    color: $main-color;
+    line-height: 18px;
+    font-weight: 700;
+  }
+
+  &__button {
+    height: 40px;
+    width: 100%;
+    border-radius: 5px;
+    font-size: 14px;
+    background-color: $blue-accent;
+    color: $color-white;
```

Comment:
Ensure that the variable `$color-white` is defined in your SCSS files or imported correctly. Otherwise, it will cause a compilation error.